client/state/themes/actions.js

```diff
@@ -609,7 +611,8 @@ function transferStatusFailure( siteId, transferId, error ) {
 export function pollThemeTransferStatus( siteId, transferId, interval = 3000, timeout = 180000 ) {
 	const endTime = Date.now() + timeout;
 	return dispatch => {
-		const pollStatus = ( resolve, reject ) => {
+		dispatch( requestAutomatedTransferStatus( siteId ) );
+		const pollStatus = resolve => {
 			if ( Date.now() > endTime ) {
 				// timed-out, stop polling
 				dispatch( transferStatusFailure( siteId, transferId, 'client timeout' ) );
```

Comment:
should we be calling `stopPolling()` here? Seems the timeout should probably be built into `automated-transfer/status/index.js`, if we think the timeout is necessary? 

My view is that the timeout is useful for giving an error if it doesn't complete by a certain time.